src/Microsoft.DotNet.Interactive/ResolvedPackageReference.cs

```diff
@@ -32,15 +31,14 @@ public class ResolvedPackageReference : PackageReference
             {
                 PackageRoot = new FileInfo(path).Directory?.Parent?.Parent?.FullName;
             }
-            Requested = requested;
         }
 
         public IReadOnlyList<string> AssemblyPaths { get; }
 
         public IReadOnlyList<string> ProbingPaths { get; }
 
         public string PackageRoot { get; }
-        internal bool Requested { get; }
+        public bool Requested { get; }
```

Comment:
you have to delete this property too, the contract tests are broken

Reply:
thanks

Reply:
removed property